clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -128,12 +128,13 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
                 middleButton={
                   <PrivacyPolicyLink experience={experienceConfig} />
                 }
+                isMobile={isMobile}
               />
-            }
+            )}
           />
         ) : null
       }
-      renderModalContent={({ onClose }) => (
+      renderModalContent={() => (
```

Comment:
if this no longer needs an argument, I think it'd be cleaner to revert it back to a component, i.e.

```js
modalContent={
  <div>
  ...
  </div>
}
```

or even to let this be `children`